content/silver/ref/expr/pattern-matching.md

```diff
@@ -121,9 +143,35 @@ end
 ```
 The relative order of patterns for forwarding and non-forwarding productions is relevant for which pattern matches.
 In this example, the `d(_)` pattern will never be reached because any trees built by the `d` production will match the `a(_)` pattern.
+This is the same as placing a more general pattern before a more specific pattern in non-forwarding pattern matching.
 To match a forwarding production, place it at the beginning of the clauses.
 It is not an error to place it later, but it is unlikely the resulting semantics are the intended ones.
```

Comment:
not a "compile-time" error?  It might be a logical error.

Reply:
I don't understand this comment.

Reply:
This was for  what is now line 148 - "It is not an error to place it later".  It is not a compile-time error but it is likely, as it says, a logical error.  The type of error it is not should be more clear.